tests/test_session.py

```diff
@@ -292,9 +291,9 @@ def test_start_transcript_file_write(new_mesh_session, tmp_path=pyfluent.EXAMPLE
 
     session.exit()
 
+    time.sleep(1)
+
     assert returned
-    if os.path.exists(file_path):
```

Comment:
This step was redundant as the entire directory was being cleaned before starting the tests.